api/src/test/java/org/pmiops/workbench/db/dao/ParticipantDaoTest.java

```diff
@@ -0,0 +1,85 @@
+package org.pmiops.workbench.db.dao;
+
+import static org.junit.Assert.*;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.pmiops.workbench.db.model.Participant;
+import org.pmiops.workbench.db.model.ParticipantKey;
+import org.pmiops.workbench.model.CohortStatus;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.liquibase.LiquibaseAutoConfiguration;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.context.annotation.Import;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.util.List;
+
+@RunWith(SpringRunner.class)
+@DataJpaTest
+@Import(LiquibaseAutoConfiguration.class)
+@AutoConfigureTestDatabase(replace= AutoConfigureTestDatabase.Replace.NONE)
+public class ParticipantDaoTest {
+
+    private Participant participant1;
+    private Participant participant2;
+    private static Long COHORT_ID = 1L;
+    private static Long CDR_VERSION_ID = 1L;
+
+    @Before
+    public void onSetup() {
+        ParticipantKey key1 = new ParticipantKey().cohortId(COHORT_ID).cdrVersionId(CDR_VERSION_ID).participantId(1);
+        ParticipantKey key2 = new ParticipantKey().cohortId(COHORT_ID).cdrVersionId(CDR_VERSION_ID).participantId(2);
+        participant1 = new Participant().participantKey(key1).status(CohortStatus.INCLUDED);
+        participant2 = new Participant().participantKey(key2).status(CohortStatus.EXCLUDED);
+        participantDao.save(participant2);
+        participantDao.save(participant1);
+    }
+
+    @After
+    public void onTearDown() {
+        participantDao.delete(participant1);
```

Comment:
I don't think this is strictly necessary... unless something has changed, there should be an empty DB at the start of each test invocation.

Reply:
If we have more than 1 test case delete is necessary, otherwise table inserts are duplicated.